code/modules/recycling/disposal/outlet.dm

```diff
@@ -122,8 +133,23 @@
 		return
 	to_chat(user, span_notice("You silently disable the sanity checking on \the [src]'s ejection force."))
 	obj_flags |= EMAGGED
+	switch(eject_speed)
```

Comment:
You don't need this, you can just set the ejection to YEET without having to check which one it's already on.